pkg/importer/dataStream.go

```diff
@@ -324,121 +306,73 @@ func (d *DataStream) http() (io.ReadCloser, error) {
 	return countingReader, nil
 }
 
-//registryData - temporary data used by import from registry flow
-//is deleted as part of dataStream.Close()
-type registryData struct {
-	dataDir string
-	file    *os.File
-}
-
-func (r registryData) Cleanup() error {
-	glog.V(1).Infof("registryData - deleting all the data")
-	if r.file != nil {
-		r.file.Close()
-	}
-	err := os.RemoveAll(r.dataDir)
-	if err != nil {
-		glog.Errorf(" Failed removing directory ", err.Error())
-	}
-	return nil
-}
-
-const (
-	//ContainerDiskImageDir - Expected disk image location in container image as described in
-	//https://github.com/kubevirt/kubevirt/blob/master/docs/container-register-disks.md
-	ContainerDiskImageDir = "disk"
-	//TempContainerDiskDir - Temporary location for pulled containerImage
-	TempContainerDiskDir = "tmp"
```

Comment:
Why to remove? the only change here must be that TempContainerDikDur must be set to scratch space location